prov/sockets/src/sock_progress.c

```diff
@@ -2464,6 +2463,11 @@ static void sock_thread_set_affinity(char *s)
 	int j, first, last,stride;
 	cpu_set_t mycpuset;
 	pthread_t mythread;
+ 	
+	if(!s) {
+		SOCK_LOG_DBG("Invalid FI_SOCKETS_PE_AFFINITY value\n");
+		return;
+	}
```

Comment:
The coverity issue deals with the saveptra variable:

CID 100409 (#1 of 1): Explicit null dereferenced (FORWARD_NULL)6. var_deref_model: Passing &saveptra to __strtok_r_1c, which dereferences null saveptra. [show details]

This is not an actual issue.  The checker just can't distinguish the first call to strtok_r, versus a subsequent call. I marked this defect as invalid in coverity.  Hopefully that works to dismiss it.